src/app/staticanalysis/modules/similarVariableNames.js

```diff
@@ -45,48 +46,21 @@ function findSimilarVarNames (vars) {
   var similar = []
   var comb = {}
   vars.map((varName1) => vars.map((varName2) => {
-    if (varName1.length > 1 && varName2.length > 1 && varName2 !== varName1 && !(comb[varName1 + ';' + varName2] || comb[varName2 + ';' + varName1])) {
+    if (varName1.length > 1 && varName2.length > 1 && varName2 !== varName1 && !isCommonPrefixedVersion(varName1, varName2) && !(comb[varName1 + ';' + varName2] || comb[varName2 + ';' + varName1])) {
       comb[varName1 + ';' + varName2] = true
-      var distance = levenshteinDistance(varName1, varName2)
+      var distance = levenshtein.get(varName1, varName2)
       if (distance <= 2) similar.push({ var1: varName1, var2: varName2, distance: distance })
     }
   }))
   return similar
 }
 
-function getFunctionVariables (contract, func) {
-  return contract.stateVariables.concat(func.localVariables)
+function isCommonPrefixedVersion (varName1, varName2) {
+  return (varName1.startsWith('_') || varName2.startsWith('_')) && (varName1.slice(1) === varName2 || varName1 === varName2.slice(1))
```

Comment:
This would return true for `x_abc` and `_abc` - do we want that? If not, it should be something like `(varName1.startsWith('_') && varName1.slice(1) == varName2) || (...)`